examples/HydroValleys/case3_5years.jl

```diff
@@ -28,32 +28,33 @@ data = HydroPowerModels.parse_folder(joinpath(WEAVE_ARGS[:testcases_dir],"case3"
 params = set_param( stages = 12*5, 
                     model_constructor_grid  = DCPPowerModel,
                     post_method             = PowerModels.post_opf,
-                    solver                  = ClpSolver())
+                    optimizer                  = Clp.Optimizer)
 
 #' ## Build Model
 m = hydrothermaloperation(data, params);
 
 #' ## Solve
-status = solve(m, iteration_limit = 60);
+status = SDDP.train(m.policygraph;iteration_limit = 100);
 
 #' ## Simulation
-srand(1111);
-results = simulate_model(m, 100);
+import Random
+Random.seed!(1111);
+results = HydroPowerModels.simulate(m, 100);
 
 #' ## Testing Results
 #' Objective
-using Base.Test
-@test isapprox(results["simulations"][1]["objective"], 59800.0, atol=1e-2)
+using Test
+@test isapprox(results[:simulations][1][1][:objective], 58528.47, atol=1)
```

Comment:
Ditto on the value.

Reply:
Changed to sum(s[:stage_objective] for s in results[:simulations][1]), but still different.